src/containers/namespace-detail/namespace-detail.tsx

```diff
@@ -314,18 +312,25 @@ export class NamespaceDetail extends React.Component<IProps, IState> {
               />
             ) : (
               <section className='body'>
-                <CollectionList
-                  updateParams={updateParams}
-                  params={params}
-                  ignoredParams={ignoredParams}
-                  collections={collections}
-                  itemCount={itemCount}
-                  showControls={this.state.showControls}
-                  handleControlClick={(id, action) =>
-                    this.handleCollectionAction(id, action)
-                  }
-                  repo={this.context.selectedRepo}
-                />
+                {
+                  <CollectionList
+                    updateParams={updateParams}
+                    params={params}
+                    ignoredParams={ignoredParams}
+                    collections={collections}
+                    itemCount={itemCount}
+                    showControls={this.state.showControls}
+                    handleControlClick={(id, action) =>
+                      this.handleCollectionAction(id, action)
+                    }
+                    repo={this.context.selectedRepo}
+                    addAlert={(alert) =>
+                      this.setState({ alerts: [...alerts, alert] })
+                    }
+                    reload={() => this.loadAll()}
+                    context={this.context}
+                  />
+                }
```

Comment:
```suggestion
                <CollectionList
                  updateParams={updateParams}
                  params={params}
                  ignoredParams={ignoredParams}
                  collections={collections}
                  itemCount={itemCount}
                  showControls={this.state.showControls}
                  handleControlClick={(id, action) =>
                    this.handleCollectionAction(id, action)
                  }
                  repo={this.context.selectedRepo}
                  addAlert={(alert) =>
                    this.setState({ alerts: [...alerts, alert] })
                  }
                  reload={() => this.loadAll()}
                  context={this.context}
                />
```

(just remoes the extra `{}` around a component)